core/src/com/fallenflame/game/FogController.java

```diff
@@ -60,9 +82,10 @@ public void updateFog(Vector2 scale) {
                 }
             }
         }
+        draw(canvas, delta);
     }
 
-    public void draw(GameCanvas canvas, float delta) {
+    private void draw(GameCanvas canvas, float delta) {
         canvas.begin();
         canvas.drawFog(fog, delta);
         canvas.end();
```

Comment:
Very very minor, but this probably doesn't need to be a separate function anymore